themes/fast-track-theme/lms/static/sass/course/course_dashboard.scss

```diff
@@ -0,0 +1,424 @@
+/* COURSE DASHBOARD - info.html */
+$ft-orange: #f76b1c;
+
+body.view-in-course .info-wrapper {
+  max-width: 1366px !important;
+  box-shadow: none !important;
+}
+
+.view-course-info #main {
+  background: #eee;
+
+  .container {
+    display: inline-block;
+    width: calc(100% - 390px);
+    background: transparent;
+
+    .info-wrapper {
+      background: transparent;
+      border: 0 none;
+    }
+  }
+
+  .updates {
+    padding-top: 0 !important;
+  }
+
+  * {
+    letter-spacing: 0;
+  }
+
+  .course-info-body div {
+    line-height: 1.5rem;
+  }
+}
+
+.sidebar-ft {
+  width: 300px;
+  margin-left: 70px;
+  display: inline-block;
+  vertical-align: top;
+  color: #777;
+
+  .home {
+    background: url("../images/welcomeBackground.jpg");
```

Comment:
We import this one into lms-main, it compiles to /css folder which is at the same level as images, to, it will work.